src/main/charts/bamboo/values.yaml

```diff
@@ -275,6 +275,14 @@ ingress:
   # pre-provisioned Ingress Controller being available.
   # 
   create: false
+
+  # -- The class name used by the ingress controller if it's being used.
```

Comment:
Looking for feedback for the documentation text in the `values.yaml` files. 🙏 

This is a relevant documentation - https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class but I didn't include it as it promotes `IngressClass` objects that are not created by default. TBH, I think it a bit confusing. K8s has introduced `ingressClassName` into ingress `spec` but `ingress.class` annotation still t[akes a preference if it is there](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#ingressspec-v1-networking-k8s-io). For example, Nginx and ALB ingress controllers are still listing `ingress.class` in their installation documentation.